website/content/docs/k8s/deployment-configurations/multi-cluster/kubernetes.mdx

```diff
@@ -81,18 +81,17 @@ Modifications:
 
 1. The Consul datacenter name is `dc1`. The datacenter name in each federated
    cluster **must be unique**.
-1. ACLs are enabled in the above config file. They can be disabled by setting:
-
+1. ACLs are enabled in the above config file. When ACLs are enabled, primary clusters must be able to make requests to the Kubernetes API URLs of secondary clusters. To disable ACLs for testing purposes, change the following settings:
```

Comment:
```suggestion
1. ACLs are enabled in the template configuration. When ACLs are enabled, primary clusters must be able to make requests to the Kubernetes API URLs of secondary clusters. To disable ACLs for testing purposes, change the following settings:
```
I am assuming that the configuration is intended to be a template and not an example.